backend/api/completions.ts

```diff
@@ -80,6 +81,7 @@ export function completionInstructions({ knex }: ApiContext) {
         .select<any, CourseTranslation[]>("instructions")
```

Comment:
This still has the problem of storing the instructions in the `course_translations` - what if the language is not Finnish or English?

Reply:
Ah I see now. I put the `instructions` in `course_translations` because the `instructions` are similar to the course `description` column and should only be displayed in the completions page as either Finnish or English (same languages as mooc.fi). I accidentally used the `completion` language (which can be any language) to select the instructions row when I should be using the active language based on the mooc.fi profile

I changed the language select to pull the language from the current langauge context. Though I didn't see an immediate way to get the country code from the frontend as well, I added a switch statement to assign the languages to their respective countries we have available in mooc.fi (I suppose this works until we need Irish English or British English, etc). Also, the instructions card is now hidden if there are no instructions in the DB.